sources/Interop/Windows/winrt/wrl/client/ComPtr`1.Manual.cs

```diff
@@ -4,150 +4,401 @@
 // Original source is Copyright © Microsoft. All rights reserved.
 
 using System;
+using System.ComponentModel;
 using System.Diagnostics;
+using System.Runtime.CompilerServices;
 using static TerraFX.Interop.Windows;
 
 namespace TerraFX.Interop
 {
-    public unsafe partial struct ComPtr<T> : IDisposable
+    /// <summary>
+    /// A type that allows working with pointers to COM objects more securely.
+    /// </summary>
+    /// <typeparam name="T">The type to wrap in the current <see cref="ComPtr{T}"/> instance.</typeparam>
+    /// <remarks>
+    /// While this type is not marked as <see langword="ref"/> so that it can also be used in fields, make sure
+    /// to keep the reference counts properly tracked if you do store <see cref="ComPtr{T}"/> instances on the heap.
+    /// </remarks>
+    public unsafe struct ComPtr<T> : IDisposable
         where T : unmanaged
     {
-        private T* ptr_;
+        /// <summary>
+        /// The raw pointer to a COM object, if existing.
+        /// </summary>
+        private T* pointer;
```

Comment:
This should remain `ptr_`. It matches the name as taken from the original source.

Reply:
Reverted in 650cef56939beacfd47c0eccfb75e500466f0f88, didn't know you wanted to keep the same name there 😄